rustls-cert-gen/src/cert.rs

```diff
@@ -0,0 +1,431 @@
+use bpaf::Bpaf;
+use rcgen::{
+	BasicConstraints, Certificate, CertificateParams, DistinguishedName, DnType,
+	DnValue::PrintableString, ExtendedKeyUsagePurpose, IsCa, KeyUsagePurpose, SanType,
+};
+use std::{fmt, fs::File, io, path::Path};
+
+#[derive(Debug, Clone)]
+/// PEM serialized Certificate and PEM serialized corresponding private key
```

Comment:
In terms of item ordering I think it would make more sense to keep `PemCertifiedKey` closer to `EndEntity` and `Certificate`, also keeping `EndEntityBuilder` and `CaBuilder` together.